charge_point_gateway_format.proto

```diff
@@ -41,6 +45,12 @@ message LogData {
 message AuthorizeRequestData { string charge_point_id = 1; }
 message AuthorizeRequestAckData { Response resp = 1; }
 
+message CheckAvailabilityRequestData {
+  string charge_point_id = 1;
```

Comment:
Could I ask why the type of charge_point_id is string but connector_id is int32?

Reply:
implementatd as stated in docs.
charge_point_id could be AlphaNumeric while connector_id is not.